src/db/postgres.ts

```diff
@@ -58,6 +58,23 @@ async function query(dbConfig: ClientConfig, sql: string): Promise<void> {
     await db.end()
 }
 
+async function queryWithParams(
+    dbConfig: ClientConfig,
+    sql: string,
+    params?: any[]
```

Comment:
question: how does this pass linting, there's rule about not allowing `any`?

Reply:
I added some exceptions, seems to be hard to parse the JSON in TypeScript because is polymorphic.